docs/Support/faq.md

```diff
@@ -26,37 +38,105 @@ The signature is stored in your browser local storage for convenience. You can c
 
 We are currently working on implementing a more direct solution for using Lit Actions and ACCs together, allowing users to define ACCs that provision access based on the result of any Lit Action. This will enable the generation of more complex ACCs and the use of off-chain data in your conditions.
+
 Before this solution has been released (ETA ~1 month) make sure you set the expiration time in the SIWE Auth Sig to something short so that the user can't take it and pass it around to others who shouldn’t have access.
 
+### 3. How to get the BTC address for a PKP?
+
+There are 2 ways of getting the BTC address associated with a PKP. First, you may get it from the [PKP Explorer](https://explorer.litprotocol.com/pkps). If you want that programmatically you may use the public key of the PKP to get the BTC address as demonstrated below:
+
+```js
+import * as bitcoinjs from "bitcoinjs-lib";
+
+if (publicKey.startsWith("0x")) {
+  publicKey = publicKey.slice(2);
+}
+pubkeyBuffer = Buffer.from(publicKey, "hex");
+
+// get the btc address from the public key
+const pkpBTCAddress = bitcoinjs.payments.p2pkh({
+  pubkey: pubkeyBuffer,
+}).address;
+```
+
+### 4. “Error: Invalid arrayify value”upon passing IPFS CID to functions (isPermittedActions, etc.) while interacting with PKPPermissions contract?
+
+The error is because the expected data type for IPFS CID in the contract is bytes. You have to use the conversion function below to convert your IPFS to bytes:
+
+```js
+function getBytesFromMultihash(ipfsId) {
+  const decoded = bs58.decode(ipfsId);
+
+  return `0x${Buffer.from(decoded).toString("hex")}`;
+}
+```
+
+### 5. How to import an npm package inside a LitAction?
 
-## Access Control
+You have to use `esbuild` to create a bundle & use that. See example [here](https://github.com/LIT-Protocol/js-serverless-function-test/tree/main/bundleTests/siwe).
+
+### 6. I’ve permitted LitAction for my PKP, do I still need to pass a valid authSig?
+
+Yes, the AuthSig is required to authenticate with the nodes. If you pass an empty object, the nodes will throw an error. You can use a global AuthSig if it’s not being used for auth hence it can be made available to all the users. Alternatively you may produce an AuthSig on the fly using hot wallet signing.
+
+### 7. What should be the value of keyType when interacting directly with the PKPNFT Contract?
+
+The value should be `2` as it represents ECDSA which supports the PKP.
+
+### 8. I want to understand how the PKP works internally.
+
+Check out our [blog post](https://spark.litprotocol.com/how-authentication-works-with-pkps/) on how authentication works with PKPs.
+
+### 9. Minting PKP on https://explorer.litprotocol.com/mint-pkp doesn’t progress?
+
+Try removing Mumbai network from your MetaMask. When you connect your wallet to this website, it will pop-up a window for adding Mumbai. Select that & it should work now.
```

Comment:
this isn't relevant now with the update to Chronicle

Reply:
@debbly i ended up removing the q&a for now as i don't think one can quite remove the chronicle network from one's metamask. thanks for catching this!